docs/config.md

```diff
@@ -208,6 +208,13 @@ A [picomatch pattern](https://github.com/micromatch/picomatch), or array of patt
 
   Force Vite to pre-bundle Svelte libraries. Setting this `true` should improve initial page load performance, especially when using large Svelte libraries. See the [FAQ](./faq.md#what-is-going-on-with-vite-and-pre-bundling-dependencies) for details of the pre-bundling implementation.
 
+### disableCompileStats
+
+- **Type** `boolean | 'dev' | 'build'`
+- **Default:** `false`
+
+  disable svelte compile statistics
```

Comment:
> added an option to disable because --logLevel warn doesn't seem to propagate to the second vite build kit kicks off and it also removes other useful logging.

I feel like we should fix this in SvelteKit instead another option. It would be nice to reduce the number of options, similar to Vite's idea.

I also wonder if the logs should be enabled via `DEBUG=`, but we could also not do that as a start until someone finds it annoying 🤔

Reply:
fixing logLevel passthrough in kit (or vite) is indeed a good idea. But being able to disable the stats logs separately still makes sense. 
The debug log is an interesting proposition, but we'd have to restructure how we debug log today and segregate that into extra categories so that you could do `DEBUG=vite-plugin-svelte:stats:*` or something to not overload the output.

I want the logs to be visible by default so that when we enable prebundleSvelteLibraries by default, users don't get surprised by the initial wait that might cause. That isn't possible with debug i think

Reply:
my feeling for now is that we should have them enabled by default. maybe I'll change my mind after experiencing it a bit more on real projects, but they seem to be pretty helpful

I'm torn on the option to disable them. I can see that being valuable, but it also possibly means we can't remove the option without a breaking change, so I somewhat wonder if we should wait until someone asks for it. But at the same time it's already implemented :shrug:

Reply:
we can move the disable option to experimental :see_no_evil:

Reply:
done.